### PR TITLE
Updated isolated workloads formatting and info for local deployment

### PR DESCRIPTION
**Pull request by ThomasK33**
Added an info box about setting `--expose-local=false` when running `vcluster create` locally.
Also added a missing yaml annotation to the last code block.

Preview link: https://deploy-preview-187--vcluster-docs-site.netlify.app/docs/vcluster/deploy/topologies/isolated-workloads

### CODE REVIEW

File: vcluster/deploy/topologies/isolated-workloads.mdx [hunk 1]
```diff
@@ -6,16 +6,34 @@ sidebar_position: 2
...
+2. vCluster deploys a [resource quota](https://kubernetes.io/docs/concepts/policy/resource-quotas/)
+as well as a [limit range](https://kubernetes.io/docs/concepts/policy/limit-range/)
+alongside the vCluster itself. This allows restricting resource consumption of
+vCluster workloads. If enabled, sane defaults for those 2 resources are chosen.
+3. vCluster deploys a [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+alongside itself that will restrict access of vCluster workloads as well as the
+vCluster control plane to other pods in the host cluster. (only works if your host
+[cluster CNI supports network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/#prerequisites))
+
+You can adjust isolation settings through helm values (vcluster.yaml). To configure
```
Comment (deniseschannon) [inline vcluster/deploy/topologies/isolated-workloads.mdx:34]: We want to move away from referencing and calling `helm values` even if that is technically what it is and call it only `vcluster.yaml. 

I'd update this to:

Adjust your isolation settings through the `vcluster.yaml` of the vCluster:

File: vcluster/deploy/topologies/isolated-workloads.mdx [hunk 2]
```diff
@@ -28,48 +46,76 @@ policies:
...
+:::
 
 <Policies/>
 
 ## Network Isolation
 
-Workloads created by vCluster will be able to communicate with other workloads in the host cluster through their cluster ips. This can be sometimes beneficial if you want to purposely access a host cluster service, which is a good method to share services between vClusters. However, you often want to isolate namespaces and do not want the pods running inside vCluster to have access to other workloads in the host cluster.
-This requirement can be accomplished by using [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) for the namespace where vCluster is installed in.
+Workloads created by vCluster will be able to communicate with other workloads in
+the host cluster through their cluster ips. This can be sometimes beneficial if
```
Comment (deniseschannon) [inline vcluster/deploy/topologies/isolated-workloads.mdx:64]: Capitalize `IPs`

File: vcluster/deploy/topologies/isolated-workloads.mdx [hunk 3]
```diff
@@ -28,48 +46,76 @@ policies:
...
+you want to purposely access a host cluster service, which is a good method to
+share services between vClusters. However, you often want to isolate namespaces
```
Comment (deniseschannon) [inline vcluster/deploy/topologies/isolated-workloads.mdx:66]: update from `vClusters` to `virtual clusters`. We should never pluralize vCluster.

File: vcluster/deploy/topologies/isolated-workloads.mdx [hunk 4]
```diff
@@ -28,48 +46,76 @@ policies:
...
+and do not want the pods running inside vCluster to have access to other workloads
+in the host cluster. This requirement can be accomplished by using [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+for the namespace where vCluster is installed in.
+
+vCluster can automatically deploy a network policy for you by enabling the
+following option:
```
Comment (deniseschannon) [inline vcluster/deploy/topologies/isolated-workloads.mdx:72]: "enabling the following option in your `vcluster.yaml`:"

File: vcluster/deploy/topologies/isolated-workloads.mdx [hunk 5]
```diff
@@ -28,48 +46,76 @@ policies:
...
-By default vCluster will create Service Account Tokens for each pod and inject them as an annotation in the respective pods
-metadata. This can be a security risk in certain scenarios. To mitigate this there's an option in vCluster
-which creates separate secrets for each pods Service Account Token and mounts it accordingly using projected volumes. This option
-is not enabled by default but can be enabled on demand. To enable you can use:
+By default vCluster will create Service Account Tokens for each pod and inject
+them as an annotation in the respective pods metadata. This can be a security
+risk in certain scenarios. To mitigate this there's an option in vCluster
+which creates separate secrets for each pods Service Account Token and mounts it
+accordingly using projected volumes. This option is not enabled by default but
+can be enabled on demand. To enable you can use:
```
Comment (deniseschannon) [inline vcluster/deploy/topologies/isolated-workloads.mdx:116]: Replace `To enable you can use:` to `To enable this option, enable the configuration in the vcluster.yaml:`

**Review by deniseschannon — CHANGES_REQUESTED** [review]
The grammar is honestly not great in this whole page, but I pointed out the obvious issues of how we want to present information to our users.

**Comment by ThomasK33** [comment]
@deniseschannon I can have it run through Grammarly if that helps.

This PR mainly includes lint fixes from [markdownlint](https://github.com/DavidAnson/markdownlint) and that one added info box for a local setup.

I updated the PR with your fixes.

**Comment by deniseschannon** [comment]
@ThomasK33 There are still notes about how it could be a security risk in the last paragraph that I would like updated. Since this is a default option, why would we have a default option that has security risks? Therefore I want it changed to not include the idea of security risks could exist.

**Pull request merged**